### ISSUE TITLE
It's not working on Sketch 52

### ISSUE THREAD

**Issue by ilustrebruno** [einancunlu/Flatten-Plugin-for-Sketch#19]
I click on Flatten and nothing happens.

**Comment by ralts00** [comment]
Please :)

**Comment by jessewallace** [comment]
Having the same problem, Flatten plugin no longer working in Sketch 52

**Comment by einancunlu** [comment]
Hello guys, sorry for the late answer. I fixed it mostly, you can find the half-working version here [Flatten.sketchplugin.zip](https://github.com/einancunlu/Flatten-Plugin-for-Sketch/files/2475557/Flatten.sketchplugin.zip). I couldn't find a way to fix the restore feature, so this feature doesn't work still.

Also, I'm afraid I have to pull off the plugin. I really don't like how Sketch team works anymore. I waited enough with the hope that they would update the plugin development documentation fully and be more careful with the updates. But this doesn't happening, so I don't have any desire left to fix the plugins every time Sketch updates. 

Sorry for that again. Hope you can find a work around soon. I will switch to Figma as soon as they have a proper plugin environment. Suggest you do the same. : )

**Comment by ralts00** [comment]
Please don't leave the ship, you are the master! :)

**Comment by illustree** [comment]
such awesome plugin ... please can you bring it alive again :___-(

**Comment by ralts00** [comment]
Yes, again, even with the park you created in sketch 52, he died in 53 :(
